srcpkgs/seer/template

```diff
@@ -0,0 +1,32 @@
+# Template file for 'seer'
+pkgname=seer
+version=1.17
+revision=1
+build_style=cmake
+build_wrksrc="src"
+hostmakedepends="pkg-config"
+makedepends="qt5-devel qt5-charts-devel"
+depends="gdb"
+short_desc="GUI frontend to gdb for Linux"
+maintainer="Orphaned <orphan@voidlinux.org>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/epasveer/seer"
+changelog="https://raw.githubusercontent.com/epasveer/seer/main/CHANGELOG.md"
+distfiles="https://github.com/epasveer/seer/archive/v${version}.tar.gz"
+checksum=085d096731d8d86a374107e36f01bb66582ac2509be4bafc2a443a4275973705
+
+case "$XBPS_TARGET_MACHINE" in
+	i686*) ;;
+	x86_64*) ;;
+	*) hostmakedepends+=" qt5-qmake qt5-host-tools" ;;
```

Comment:
This looks wrong, you are probably trying to target cross builds, not specific architectures which can be compiled either native or cross.

```
if [ "$CROSS_BUILD" ]; then
	hostmakedepends+=" qt5-qmake qt5-host-tools"
fi
```